checklist.md

```diff
@@ -38,14 +39,18 @@ BAD example:
 
 3. [STYLES] - Get used to style all elements using classes. And don't increase
 selectors specificity unless completely necessary.
-4. [STYLES] - If you need to adjust positioning with `1px` or fractional numbers
-like `0.5`, you are doing something wrong, change the alignment technique you use
+
+4. [STYLES] - If you need to adjust positioning with `1px` or use fractional numbers
+like `0.5` ANYWHERE, you are doing something wrong, change the alignment technique you use
+
 5. [STYLES] - Be consistent with your margins - if you have many sections in a
 row, add margins either to the bottom or to the top so that it will be easier
 to identify how to position the next element
+
 6. [BEM] - Check your BEM structure using BEM-linter (`npm run lint`) and
-[this list](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes)
-7. [BEM] - Make sure to follow BEM naming convention
+[this list](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes).
+Also, make sure to follow BEM naming convention:
```

Comment:
Let's separate this points:
1. checking BEM using BEM-linter
2. Our typical bem mistakes guide

Don't combine several requirements in a one point, it's much harder to follow such requirements